### PR TITLE
feat(hpc-vmware-managed-vcd): add activate license

### PR DESCRIPTION
**Pull request by pauldkn**
ref: MANAGER-14741

| Question         | Answer
| ---------------- | ---
| Branch?          | `hpc-vcd-beta`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | feat#MANAGER-14741
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-options-tile/OrganizationOptionsTile.component.tsx [hunk 1]
```diff
@@ -44,3 +65,4 @@ export default function OrganizationOptionsTile({
     </div>
   );
 }
+// orderconfirmation.tsx // order funnel example
```
Comment (darsene) [inline packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-options-tile/OrganizationOptionsTile.component.tsx:68]: To remove

File: packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-options-tile/OrganizationOptionsTile.component.tsx [hunk 2]
```diff
@@ -9,11 +13,20 @@ import { ODS_ICON_NAME } from '@ovhcloud/ods-components';
...
+  organizationName: string;
+}>) {
   const { t } = useTranslation('dashboard');
 
+  const orderBaseUrl = useOrderURL('express_review_base');
+  const vcdProductSettings = getVcdProductSettings(organizationName);
+  const orderLink = `${orderBaseUrl}?products=~(${vcdProductSettings})`;
+  const handleActivateLicense = () => {
+    window.open(orderLink, '_blank', 'noopener,noreferrer');
+  };
```
Comment (darsene) [inline packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-options-tile/OrganizationOptionsTile.component.tsx:28]: Could be split in a dedicated hook

File: packages/manager/modules/order/src/order.constant.ts [hunk 3]
```diff
@@ -33,6 +33,15 @@ export const getKMSProductSettings = ({ region }: { region: string }) =>
...
   });
 
+export const getVcdProductSettings = (serviceName: string) =>
+  JSURL.stringify({
+    serviceName,
+    productId: 'vmware-cloud-director',
+    planCode: 'vcd-microsoft-license',
+    duration: 'P1M',
+    pricingMode: 'default',
+  });
```
Comment (darsene) [inline packages/manager/modules/order/src/order.constant.ts:43]: To rework regarding PU feedback about arguments to provide

**Review by chipp972 — APPROVED** [review]

File: packages/manager/apps/hpc-vmware-managed-vcd/src/data/hooks/useManagedVcdOrganization.ts [hunk 4]
```diff
@@ -16,11 +21,17 @@ const getVcdOrganizationBackupQueryKey = (id: string) => [
...
+}: IUseManagedVcdOrganization) => {
   return useQuery<ApiResponse<IVcdOrganization>, ApiError>({
     queryKey: getVcdOrganizationQueryKey(id),
     queryFn: () => getVcdOrganization(id),
     retry: false,
+    refetchInterval,
+    refetchOnWindowFocus,
     ...{
       keepPreviousData: true,
     },
```
Comment (tibs245) [inline packages/manager/apps/hpc-vmware-managed-vcd/src/data/hooks/useManagedVcdOrganization.ts:37]:
```suggestion
   keepPreviousData: true,
```

Comment (pauldkn) [reply packages/manager/apps/hpc-vmware-managed-vcd/src/data/hooks/useManagedVcdOrganization.ts]: Thanks for pointing this out, I've updated it according to the tanstack-query v5 docs (keepPreviousData option removed)

**Review by darsene — APPROVED** [review]

**Pull request merged**